src/contracts/mocks/testnet-helpers/TestnetRWAERC20.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {TestnetERC20} from 'src/contracts/mocks/testnet-helpers/TestnetERC20.sol';
+
+contract TestnetRWAERC20 is TestnetERC20 {
+  mapping(address holder => bool isAuthorized) public authorized;
+
+  modifier onlyAuthorized(address potentialHolder) {
+    require(authorized[potentialHolder], 'UNAUTHORIZED_RWA_HOLDER');
+    _;
+  }
+
+  constructor(
+    string memory name,
+    string memory symbol,
+    uint8 decimals,
+    address owner
+  ) TestnetERC20(name, symbol, decimals, owner) {}
+
+  function authorize(address holder) public {
```

Comment:
add bool so we can remove auth too

Reply:
[78c0cfe](https://github.com/aave/aave-v3-horizon/pull/1/commits/78c0cfe181273d953b2a026a5bbbb2dfa152068a)